bin/proving-service/src/api/prover.rs

```diff
@@ -77,8 +84,16 @@ impl ProverRpcApi {
 
         // Record the transaction_id in the current tracing span
         let transaction_id = proof.id();
+        let duration = start_time.elapsed();
         tracing::Span::current().record("id", tracing::field::display(&transaction_id));
 
+        info!(target: COMPONENT,
+            proof_type = "transaction",
+            id = %transaction_id,
+            duration_ms = %duration.as_millis(),
+            "Proof generated successfully"
+        );
```

Comment:
We could just rely on the span close event for the duration instead of doing it manually. Just need to make sure this fn has `#[instrument]` annotation.